homeassistant/components/tessie/entity.py

```diff
@@ -0,0 +1,55 @@
+"""Tessie parent entity class."""
+
+
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN, TessieApi
+
+MODELS = {
+    "model3": "Model 3",
+    "modelx": "Model X",
+    "modely": "Model Y",
+    "models": "Model S",
+}
```

Comment:
This probably belongs in const

Reply:
This is probably opinion, but does it below on const if this is the only file that will use it? Not a bad idea regardless.

Reply:
Moved